web/settings.platformsh.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @file
+ * Platform.sh Backdrop CMS configuration file.
+ */
+
+// Configure relationships.
+if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
+  $relationships = json_decode(base64_decode($_ENV['PLATFORM_RELATIONSHIPS']), TRUE);
+  if (empty($databases['default']) && !empty($relationships)) {
+    foreach ($relationships as $key => $relationship) {
+      $drupal_key = ($key === 'database') ? 'default' : $key;
+      foreach ($relationship as $instance) {
+        if (empty($instance['scheme']) || ($instance['scheme'] !== 'mysql' && $instance['scheme'] !== 'pgsql')) {
+          continue;
+        }
+        $db = [
+          'driver' => $instance['scheme'],
+          'database' => $instance['path'],
+          'username' => $instance['username'],
+          'password' => $instance['password'],
+          'host' => $instance['host'],
+          'port' => $instance['port'],
+        ];
+        if (!empty($instance['query']['compression'])) {
+          $database['pdo'][PDO::MYSQL_ATTR_COMPRESS] = TRUE;
+        }
+        if (!empty($instance['query']['is_master'])) {
+          $databases[$drupal_key]['default'] = $db;
+        }
+        else {
+          $databases[$drupal_key]['slave'][] = $db;
+        }
+      }
+    }
+  }
+}
+
+// Enable multibyte support in the database:
+$database_charset = 'utf8mb4';
+
+// The 'trusted_hosts_pattern' setting allows an admin to restrict the Host header values
+// that are considered trusted.  If an attacker sends a request with a custom-crafted Host
+// header then it can be an injection vector, depending on how the Host header is used.
+// However, Platform.sh already replaces the Host header with the route that was used to reach
+// Platform.sh, so it is guaranteed to be safe.  The following line explicitly allows all
+// Host headers, as the only possible Host header is already guaranteed safe.
+$settings['trusted_host_patterns'] = ['.*'];
+
+// Configure private and temporary file paths.
+if (isset($_ENV['PLATFORM_APP_DIR'])) {
+  if (!isset($settings['file_private_path'])) {
+    $settings['file_private_path'] = $_ENV['PLATFORM_APP_DIR'] . '/private';
+  }
+  if (!isset($settings['file_temporary_path'])) {
+    $settings['file_temporary_path'] = $_ENV['PLATFORM_APP_DIR'] . '/tmp';
+  }
+}
+
+// Import variables prefixed with 'backdrop:' into $conf.
+if (isset($_ENV['PLATFORM_VARIABLES'])) {
+  $variables = json_decode(base64_decode($_ENV['PLATFORM_VARIABLES']), TRUE);
+  $prefix_len = strlen('backdrop:');
+  $backdrop_globals = array('cookie_domain', 'installed_profile', 'drupal_hash_salt', 'base_url');
```

Comment:
this should probably be `backdrop_hash_salt`

Reply:
Actually, I checked on this and in Backdrop it's not a global. For us it's `$settings['hash_salt']` and is set in `settings.php`.

Reply:
There's already code for setting the hash_salt further down the file so I just removed it from here.